client/web/src/search/results/progress/StreamingProgressCount.tsx

```diff
@@ -29,7 +30,7 @@ export const StreamingProgressCount: React.FunctionComponent<
     <>
         <small
             className={classNames(
-                'd-flex align-items-center',
+                'd-flex align-items-center px-0',
```

Comment:
```suggestion
                'd-flex align-items-center',
```

![Nov-11-2021 14-20-35](https://user-images.githubusercontent.com/9516420/141313867-db54ec6e-d6e4-422e-9d51-c0c9b86e9148.gif)

Let's keep this padding so the layout doesn't shift between the loading and loaded states